packages/mod.go

```diff
@@ -33,9 +33,10 @@ type Mod struct {
 
 func (m Mod) ExtractPurlsFromManifest() (purls []string) {
 	for _, s := range m.ProjectList.Projects {
-		version := strings.Replace(s.Version, "v", "", -1)
-
-		if len(version) > 0 { // There must be a version we can use
+		if len(s.Version) > 0 { // There must be a version we can use
+			// OSS Index no likey v before version, IQ does though, comment left so I will never forget
```

Comment:
+1, the "why" matters.

Reply:
Yeah, this is the only time I've ever felt the need to leave an explicit comment.